js/rpg_core/ImageCache.js

```diff
@@ -0,0 +1,124 @@
+function ImageCache(){
+    this.prototype.initialize.call(this);
```

Comment:
OK i read all and I found there is no exception about that. I'll follow the style.